app/Jasonette/JasonViewController.m

```diff
@@ -1302,8 +1298,9 @@ - (void)setupHeader: (NSDictionary *)body{
                         self.searchController.searchBar.returnKeyType = UIReturnKeySearch;
                         self.searchController.searchBar.barTintColor = backgroundColor;
                         self.searchController.searchBar.tintColor = color;
-                        self.searchController.searchBar.backgroundColor = backgroundColor;
+                        //self.searchController.searchBar.backgroundColor = backgroundColor;
```

Comment:
Let's remove this line if it's not needed anymore

Reply:
sure.